packages/kbn-openapi-bundler/src/openapi_bundler.ts

```diff
@@ -56,22 +71,21 @@ export const bundle = async ({
 
   logger.success(`Processed ${bundledDocuments.length} schemas`);
 
-  const blankOasFactory = (oasVersion: string, apiVersion: string) =>
+  const blankOasDocumentFactory = (oasVersion: string, apiVersion: string) =>
     createBlankOpenApiDocument(oasVersion, {
-      version: apiVersion,
-      title: options?.specInfo?.title ?? 'Bundled OpenAPI specs',
-      ...omitBy(
-        {
-          description: options?.specInfo?.description,
-          termsOfService: options?.specInfo?.termsOfService,
-          contact: options?.specInfo?.contact,
-          license: options?.specInfo?.license,
-        },
-        isUndefined
-      ),
+      info: prototypeDoc?.info
+        ? { ...DEFAULT_INFO, ...prototypeDoc.info, version: apiVersion }
```

Comment:
Shouldn't `version` be overridable by the `prototypeDoc.info.version`?

Reply:
`bundle` utility groups OpenAPI specs by `info.version`. `blankOasDocumentFactory` constructs a blank OpenAPI document with a desired `info.vesion`.

`merge` utility on the other hand should produce result bundles with specified version.